packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -13,7 +13,7 @@ import '../ui/flutter/label.dart';
 import 'scaffold.dart';
 import 'theme.dart';
 
-const tooltipWait = Duration(milliseconds: 500);
+const Duration tooltipWait = Duration(milliseconds: 500);
```

Comment:
can we remove this type?

Reply:
Is that the style? I can remove, but I'd assumed that API has types (and within a method we use `var` and `final`).

Reply:
(done)